testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BuildMultiprocessTestPlayer.cs

```diff
@@ -56,6 +56,66 @@ public static void DeleteBuild()
             }
         }
 
+        [MenuItem(MultiprocessBaseMenuName + "/Windows Standalone Player")]
+        public static void BuildWindowsStandalonePlayer()
+        {
+            SaveBuildInfo(new BuildInfo() { BuildPath = BuildPath });
+
+            var buildPathToUse = BuildPath;
+            buildPathToUse += ".exe";
+
+            var buildPlayerOptions = new BuildPlayerOptions();
+            buildPlayerOptions.scenes = new[] { "Assets/Scenes/MultiprocessTestScene.unity" };
+            buildPlayerOptions.locationPathName = buildPathToUse;
+            buildPlayerOptions.target = BuildTarget.StandaloneWindows64;
+            buildPlayerOptions.options = BuildOptions.StrictMode |
+                BuildOptions.IncludeTestAssemblies;
+
+            BuildReport report = BuildPipeline.BuildPlayer(buildPlayerOptions);
+            BuildSummary summary = report.summary;
+
+            if (summary.result == BuildResult.Succeeded)
+            {
+                Debug.Log("Build succeeded: " + summary.totalSize + " bytes");
+                Debug.Log($"ZZZ LOCATION: {summary.outputPath}");
+            }
+
+            if (summary.result == BuildResult.Failed)
+            {
+                throw new Exception("Build failed");
+            }
+        }
+
+        [MenuItem(MultiprocessBaseMenuName + "/Build OSX")]
+        public static void BuildOSX()
+        {
+            var buildPlayerOptions = new BuildPlayerOptions();
+            var buildPathToUse = BuildPath + ".app";
+            Debug.Log(buildPathToUse);
+            buildPlayerOptions.scenes = new[] { "Assets/Scenes/MultiprocessTestScene.unity" };
+            buildPlayerOptions.locationPathName = buildPathToUse;
+            buildPlayerOptions.target = BuildTarget.StandaloneOSX;
+            buildPlayerOptions.options = BuildOptions.Development |
+                BuildOptions.AllowDebugging |
```

Comment:
why is debugging set for mac, but not windows?
and why is it set at all? (some multiprocess tests will be used for performance testing, having debug options would falsify results)